### PR TITLE
Add a way to download per_instance_metrics.json files from CRFM website

### PR DESCRIPTION
**Pull request by deepakn94**

### CODE REVIEW

File: scripts/scrape_mercury_benchmarking_efficiency_data.py [hunk 1]
```diff
@@ -75,14 +90,19 @@ def main(args):
     ]
 
     runs = scrape()
-    data = parse(runs, keys)
-    save(data, keys, args.output_file)
+    if args.save_aggregated_metrics:
+        data = parse(runs, keys)
+        save(data, keys, args.output_file)
+    if save_per_instance_metrics:
```
Comment (santhnm2) [inline scripts/scrape_mercury_benchmarking_efficiency_data.py]: `args.save_per_instance_metrics`?

File: scripts/scrape_mercury_benchmarking_efficiency_data.py [hunk 2]
```diff
@@ -49,15 +50,29 @@ def parse(runs, keys):
             all_data.append((run, *[data[key] for key in keys]))
         except Exception as e:
             print(f"Could not parse metrics for run {run} at {url}")
-            print(r.text)
             continue
 
     return all_data
 
 
+def save_per_instance_metrics(runs):
```
Comment (santhnm2) [inline scripts/scrape_mercury_benchmarking_efficiency_data.py]: Maybe to match this format we should combine the `parse` and `save` functions into a single `parse_and_save_aggregated_metrics` function or something like that

Comment (santhnm2) [reply scripts/scrape_mercury_benchmarking_efficiency_data.py]: Or add a third command line argument for `--parse_aggregated_metrics`, and then `--save_aggregated_metrics` will just download the files without parsing as in `save_per_instance_metrics`

**Comment by deepakn94** [comment]
Going to merge this for now, will add an issue to clean this up later.

**Pull request merged**